MFE/devices/src/view/createDevice/steps/SummaryStep/SummaryStep.jsx

```diff
@@ -26,6 +28,11 @@ const SummaryStep = ({
     setDeviceName('');
   };
 
+  const downloadCertificateAndKeys = () => {
+    const blob = b64ToBlob(selectedCertificate.certificateAndKeys, 'application/zip');
+    fileSaver.saveAs(blob, `${t('summaryStep.certificateAndKeys')}.zip`);
```

Comment:
I'm thinking if `URL.createObjectURL()` could replace `file-saver`. 

https://developer.mozilla.org/pt-BR/docs/Web/API/URL/createObjectURl#browser_compatibility

I think that worth the try.